src/services/likeVideoService.js

```diff
@@ -0,0 +1,50 @@
+import axios from "axios";
+
+export const addToLikeVideo = async (dataDispatch, video) => {
+  const token = localStorage.getItem("token");
+  if (token) {
+    const token = localStorage.getItem("token");
+
+    try {
+      const {
+        data: { likes },
+      } = await axios.post(
+        "/api/user/likes",
+        { video },
+        {
+          headers: {
+            authorization: token,
+          },
+        }
+      );
+      dataDispatch({ type: "ADD_TO_LIKE", payload: likes });
+    } catch (error) {
+      console.log(error);
+    }
+  } else alert("Please login first");
+};

```

Comment:
just used any react toast library for show alert

Reply:
yes, will add toast in the next PR.